packages/rs-drive-abci/src/execution/platform_events/withdrawals/build_untied_withdrawal_transactions_from_documents/v0/mod.rs

```diff
@@ -70,6 +65,7 @@ where
                     ))
                 })?;
 
+            // TODO: Use constant and double check
```

Comment:
Use constant and double check the size

Reply:
https://github.com/dashpay/rust-dashcore/pull/34

Reply:
https://github.com/dashpay/rust-dashcore/pull/34